CMakeLists.txt

```diff
@@ -119,12 +119,18 @@ endif ()
 
 set (WAMR_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
-include (${WAMR_ROOT_DIR}/build-scripts/runtime_lib.cmake)
-
-set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wformat -Wformat-security -Wshadow -Wno-unused-parameter -fvisibility=hidden")
-# set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wconversion -Wsign-conversion")
+if (MSVC)
+  include (${WAMR_ROOT_DIR}/build-scripts/runtime_lib.cmake)
```

Comment:
`runtime_lib.cmake` should always be included, don't wrap it with `if (MSVC) .. endif ()`.

Reply:
fixed